content/docs/apps/govcloud.md

```diff
@@ -83,18 +81,15 @@ Any labor involved from your project team is the responsibility of your project.
 
 #### Process
 
-1. Fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1).
-    * Specify that the org should be created in `GovCloud`.
-    * An admin will confirm the information, and [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you.
+1. [Get access to the GovCloud environment](https://account.fr.cloud.gov/signup).
+1. Request the creation of your org in the GovCloud environment:
+    * If you are in 18F/TTS at GSA, fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1). Specify that the org should be created in `GovCloud`. An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should happen within one business day.
+    * For everyone else, one of your org managers should email [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov?subject=[GovCloud Org Request]) to request the creation of your org in GovCloud. Please include "[GovCloud Org Request]" in the subject heading and specify the name of your current East/West org.
```

Comment:
You can provide a default body for the `mailto:` URL that prompts them for any necessary information as they fill out the mail.